### ISSUE TITLE
Debian 12 CIS SCA is based on the generic Debian Family benchmark

### ISSUE THREAD

**Issue by Nihlus** [wazuh/wazuh#22703]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.7.3 | Wazuh Dashboard | Manager | Packages | Debian 12 |

## Description
The SCA for Debian 12 bundled with Wazuh is currently based on the generic [Debian Family benchmark](https://www.cisecurity.org/benchmark/debian_family), resulting in a number of false positives and missing rules. There exists a [dedicated benchmark for Debian 12](https://www.cisecurity.org/benchmark/debian_linux) (12.5 at time of writing) that is more recent, more specialized, and provides more accurate rules for this version of the operating system.

## Examples
One example of a false-positive is the rule regarding shadowed password hashes, which requires SHA-512 in the generic benchmark (CIS Debian Family Linux Benchmark v1.0.0 - 09-03-2020 5.3.4) but allows for either SHA-512 or yescrypt in the dedicated benchmark (CIS Debian Linux 12 Benchmark v1.0.0 - 02-28-2024 5.3.3.4.3). yescrypt is the default in new Debian 12 installations, resulting in a false positive from the older rule.

## Justification
Other SCAs in Wazuh (such as the ones for RHEL) are seemingly based on the dedicated benchmarks instead of the generic OS family benchmark, suggesting that this is an oversight and not intentional.

## Suggested Changes
The SCA for Debian 12 shipped with Wazuh (merged in #17325) should be updated to use and reflect the Debian 12 benchmark instead of the generic OS family benchmark.

**Comment by termau** [comment]
yes having the same issue 2 years later, loads of false positives - clearly Wazuh dont want to fix them - what do I do. I can patch the isues myself but that seems like cheating

**Comment by krishz-kishore** [comment]
Did anyone managed to create a custom SCA without false positives ?

**Comment by mbsouth** [comment]
> Did anyone managed to create a custom SCA without false positives ?

We currently have half a dozen Debian 12 mail server stacks running here. 
A custom SCA would be very helpful.
